stock_reception_screen/models/stock_move_line.py

```diff
@@ -10,6 +10,11 @@ class StockMoveLine(models.Model):
     lot_life_date = fields.Datetime(
         string="End of Life Date", related="lot_id.life_date"
     )
+    original_location_dest_id = fields.Many2one(
+        comodel_name="stock.location",
+        ondelete="restrict",
+        string="Original default destination",
+    )
```

Comment:
Do you really need this field on move line? It seems used only for this screen.
Can't you have a mapping in for the reception screen only?
See below.

Reply:
After some checks, I rewrote the implementation of this, field removed.